src/routes/article.js

```diff
@@ -12,6 +12,7 @@ router.locals.routes = routes;
 
 const controller = new ArticleController();
 
+namedRouter.get("articleNew", "/new", controller.new);
```

Comment:
Maybe we should change the route name if we expand this page functionality

Reply:
Any opinions on what name to use?

Reply:
I think we can have two routes (/new and /edit/:id) for one controller (it can be named something with the words "write", "edit" or "editor")

Reply:
We need to decide how we're going to handle PUT requests.
Send it through POST and handle it there or by using a middleware like [method-override](https://github.com/expressjs/method-override)